features/0023-did-exchange/README.md

```diff
@@ -321,7 +185,7 @@ When a `request` responds to an implicit invitation, its `~thread.pthid` MUST co
 {
   "@id": "a46cdd0f-a2ca-4d12-afbf-2e78a6f1f3ef",
   "@type": "https://didcomm.org/didexchange/1.0/request",
-  "~thread": { "pthid": "did:example:21tDAKCERh95uGgKbJNHYp#invitation" },
+  "~thread": { "pthid": "did:example:21tDAKCERh95uGgKbJNHYp#didcomm" },
```

Comment:
Shouldn't this just be a UUID vs. a DID?  It comes from the @id of the OOB invitation.  Also, should the `thid` be added as the `@id` of this message, since the `~thread` is present?  Not sure, so thought I would ask.

Reply:
No, it's not `thid`; it's `pthid`. We do it that way so multiple responses from different parties can all reference the same thing. They can't do that if it's `thid`, or they'd all be part of the same conversation. They can only do it if it's `pthid`. The invitation is the parent; the new handshake protocol that results is a child of the invitation, not an inline continuation of it.

Reply:
I'm not questioning what is there --- yes that should be the "pthid".

What I'm asking is a `thid` should be added.   I think it is odd to have a `~thread` with no `thid`.  The value would be exactly the same as `@id` above.

Reply:
Ah. Sorry I missed the nuance. Seems reasonable -- and more than that, it may actually be required when a ~thread decorator is used.